### PR TITLE
Update basic-sql-operations.md

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

split to DDL DML DQL DCL sections.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: [pingcap/docs-cn#3215](https://github.com/pingcap/docs-cn/pull/3215)
- Other reference link(s): [pingcap/docs-cn#3263](https://github.com/pingcap/docs/pull/3263)

### CODE REVIEW

File: basic-sql-operations.md [hunk 1]
```diff
@@ -178,25 +160,27 @@ To show all the indexes in a table, use the `SHOW INDEX` statement:
...
 
 ```sql
-ALTER TABLE person DROP INDEX person_num;
+ALTER TABLE person DROP INDEX person_unique_id;
 ```
 
-## Insert, select, update, and delete data
+> **Note:**
+> 
+> DDL operations are not transactions. You don't need to correspond COMMIT statements when executing DDL.
```
Comment (ran-huang) [inline basic-sql-operations.md:179]:
```suggestion
> DDL operations are not transactions. You don't need to run a `COMMIT` statement when executing DDL operations.
```

File: basic-sql-operations.md [hunk 2]
```diff
@@ -206,71 +190,70 @@ To insert data into a table, use the `INSERT` statement:
...
+To view the data in a table, use the `SELECT` statement:
 
 {{< copyable "sql" >}}
 
 ```sql
-DELETE FROM person WHERE number=1;
+SELECT * FROM person;
 ```
 
+Add the column name to be queried after the `SELECT` statement:
```
Comment (ran-huang) [inline basic-sql-operations.md:233]:
```suggestion
To query a specific column, add the column name after the `SELECT` keyword:
```
Please update other descriptions as well.

**Comment by ireneontheway** [comment]
PTAL @ran-huang

**Pull request merged**